### PR TITLE
Create MCONamespace constant

### PR DESCRIPTION
**Pull request by mkenigs**
Created MCONamespace constant and used in all *.go files except for
test/helpers/utils.go which would create a cyclic import

Pulling out of https://github.com/openshift/machine-config-operator/pull/2987
Can't reopen https://github.com/openshift/machine-config-operator/pull/2982

### CODE REVIEW

**Comment by kikisdeliveryservice** [comment]
would a nocyclo solve the utils issue? for ex: https://github.com/openshift/machine-config-operator/commit/0c555186774b89964c3a6156095107a35c9d5359

**Comment by mkenigs** [comment]
I think it's a compilation issue not a linting issue:
```
$ make test-unit
CGO_ENABLED=0 go test -tags="containers_image_openpgp exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_ostree_stub" -count=1 -v ./cmd/... ./pkg/... ./lib/...
# github.com/openshift/machine-config-operator/pkg/controller/common
package github.com/openshift/machine-config-operator/pkg/controller/common
	imports github.com/openshift/machine-config-operator/test/helpers: import cycle not allowed in test
```

**Comment by mkenigs** [comment]
Didn't look into eliminating the cycle

**Comment by kikisdeliveryservice** [comment]
Gotcha :+1: 

/approve

**Comment by kikisdeliveryservice** [comment]
> Didn't look into eliminating the cycle

My suggestion was bogus :laughing: - the cycle is bc:
controller/common/helpers_test.go imports "github.com/openshift/machine-config-operator/test/helpers"

File: pkg/controller/common/constants.go [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package common
 
 const (
+	MCONamespace = "openshift-machine-config-operator"
```
Comment (kikisdeliveryservice) [inline pkg/controller/common/constants.go:4]: following conventions of the file can we add a comment for this new const?

Comment (mkenigs) [reply pkg/controller/common/constants.go:4]: Hmm yeah I always struggle coming up with a non-redundant comment for something like this. Do I have to explain what a namespace is?

Comment (cgwalters) [reply pkg/controller/common/constants.go:4]: Maybe `// MCONamespace is the namespace that should be used for all API objects owned by the MCO by default` ?

Comment (cgwalters) [reply pkg/controller/common/constants.go:4]: (In Rust¹ it's not required that the documentation comments duplicate the name of the value, which helps avoid redundancy)


¹ yes, yes 😉

**Comment by cgwalters** [comment]
Overall looks sane to me but will defer final lgtm to @kikisdeliveryservice since she did more review.

File: pkg/controller/common/constants.go (hunk 1 above)
Comment (mkenigs) [reply pkg/controller/common/constants.go:5]: Added the comment

**Comment by kikisdeliveryservice** [comment]
ok let's try our new cherrypicking process!

/lgtm

**Pull request merged**